BioMed.Api/BioMed.Api/Controllers/AuthentocationController.cs

```diff
@@ -0,0 +1,68 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using Microsoft.IdentityModel.Tokens;
+using System.IdentityModel.Tokens.Jwt;
+using System.Security.Claims;
+using System.Text;
+
+namespace BioMed.Api.Controllers
+{
+    [Route("auth")]
+    [ApiController]
+
+    public class LoginRequest
```

Comment:
Isn't it efficient to put the class in another file?

Reply:
Actually it's not necessary, I fixed it, you can look at the beginning

Reply:
ok